mesa/space.py

```diff
@@ -415,6 +412,25 @@ def exists_empty_cells(self) -> bool:
         return len(self.empties) > 0
 
 
+class Grid(BaseGrid[GridContent]):
+    @staticmethod
+    def default_val() -> GridContent:
+        """ Default value for new cell elements. """
+        return None
+
+    def _place_agent(self, pos: Coordinate, agent: Agent) -> None:
+        """ Place the agent at the correct location. """
+        x, y = pos
+        self.grid[x][y] = agent
+        self.empties.discard(pos)
+
+    def _remove_agent(self, pos: Coordinate, agent: Agent) -> None:
+        """ Remove the agent from the given location. """
+        x, y = pos
+        self.grid[x][y] = None
```

Comment:
Wait, why is this not `self.default_val()`?

Reply:
Hmm it probably should be, but I just copied the function here, so this seems to be an already present bug (and unrelated to this PR)